src/main.cpp

```diff
@@ -388,7 +389,7 @@ int main(int argc, char **argv) {
                 }
 #ifdef SINGLETHREAD
                 chain.process(ed);
-#else
+#break;
```

Comment:
This breaks the logic when compiled for multi threading since `tpool.submitEdit(ed);` will not be called.